theano/sandbox/gpuarray/blas.py

```diff
@@ -8,8 +8,8 @@
 from theano.tensor.opt import in2out
 
 from .basic_ops import as_gpuarray_variable, infer_context_name
-
 from .opt_util import inplace_allocempty
+from .type import GpuArrayType
```

Comment:
done

On Tue, Jan 19, 2016 at 1:23 PM, abergeron <notifications@github.com> wrote:

> In theano/sandbox/gpuarray/blas.py
> <https://github.com/Theano/Theano/pull/3881#discussion_r50151659>:
>
> >  from .opt_util import inplace_allocempty
> > +from .type import GpuArrayType
>
> You should also remove this since it's not used.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Theano/Theano/pull/3881/files#r50151659>.
>